purchase_request_analytic/models/purchase_request.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class PurchaseRequest(models.Model):
+    _inherit = 'purchase.request'
+
+    analytic_account_id2 = fields.Many2one(
+        comodel_name='account.analytic.account',
+        help='Use to store the value of analytic_account if there is no lines')
+    analytic_account_id = fields.Many2one(
+        compute='_compute_analytic_account_id',
+        inverse='_inverse_analytic_account_id',
+        comodel_name='account.analytic.account',
+        string='Contract / Analytic', readonly=True,
```

Comment:
Why not use string='Analytic Account' like on other places in Odoo?

Reply:
done!